folium/plugins/draw.py

```diff
@@ -50,26 +53,36 @@ class Draw(JSCSSMixin, MacroElement):
               draw: {{ this.draw_options|tojson }},
               edit: {{ this.edit_options|tojson }},
             }
-            // FeatureGroup is to store editable layers.
-            var drawnItems_{{ this.get_name() }} = new L.featureGroup().addTo(
-                {{ this._parent.get_name() }}
-            );
+            {%- if this.feature_group  %}
+                var drawnItems_{{ this.get_name() }} =
+                    {{ this.feature_group.get_name() }};
+            {%- else %}
+                // FeatureGroup is to store editable layers.
+                var drawnItems_{{ this.get_name() }} =
+                    new L.featureGroup().addTo(
+                        {{ this._parent.get_name() }}
+                    );
+            {%- endif %}
+
             options.edit.featureGroup = drawnItems_{{ this.get_name() }};
             var {{ this.get_name() }} = new L.Control.Draw(
                 options
             ).addTo( {{this._parent.get_name()}} );
-            {{ this._parent.get_name() }}.on(L.Draw.Event.CREATED, function(e) {
-                var layer = e.layer,
-                    type = e.layerType;
-                var coords = JSON.stringify(layer.toGeoJSON());
-                {%- if this.show_geometry_on_click %}
-                layer.on('click', function() {
-                    alert(coords);
-                    console.log(coords);
-                });
-                {%- endif %}
-                drawnItems_{{ this.get_name() }}.addLayer(layer);
-             });
+            {{ this._parent.get_name() }}.on(
+                    L.Draw.Event.CREATED,
+                    function(e) {
+                        var layer = e.layer,
+                            type = e.layerType;
+                        var coords = JSON.stringify(layer.toGeoJSON());
+                        {%- if this.show_geometry_on_click %}
+                        layer.on('click', function() {
+                            alert(coords);
+                            console.log(coords);
+                        });
+                        {%- endif %}
+                        drawnItems_{{ this.get_name() }}.addLayer(layer);
+                    }
+            );
```

Comment:
please don't mix styling changes with functional changes. Seems this code didn't change functionally, so don't include it in this PR then please.

Reply:
Reverted.